tests/Replication2/Mocks/FakeReplicatedLog.h

```diff
@@ -155,6 +155,12 @@ struct DelayedFollowerLog : replicated_log::AbstractFollower,
 struct TestReplicatedLog : replicated_log::ReplicatedLog {
   using ReplicatedLog::becomeLeader;
   using ReplicatedLog::ReplicatedLog;
+  TestReplicatedLog()
```

Comment:
It's used in some of the tests that are currently disabled. I assume we won't need it in the future, because the tests will probably be rewritten or replaced without using it, but I'd rather keep it until then so it's easier to follow what the old tests did.